templates/example_pairwise_spatial_enrichment.ipynb

```diff
@@ -224,7 +224,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    pheno_titles = all_data["cell_meta_cluster"].drop_duplicates()
</pre><p>I think it would be a good idea to refer to this as "settings.CELL_TYPE" instead of having the user hard-code it in. Same thing earlier in the notebook, when "segmentation_label" is hard-coded in. Probably better to have that as "settings.CELL_LABEL".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
'segmentation_label' references the existing column name in the array for the segmented tiff images so that one is necessary, but I the you're totally right that 'cell_meta_cluster' should not be hard-coded!